sdk/core/Azure.Core/tests/OperationInternalTests.cs

```diff
@@ -18,7 +18,7 @@ namespace Azure.Core.Tests
     public class OperationInternalTests
     {
         private static readonly string DiagnosticNamespace = "Azure.Core.Tests";
-        private static readonly ClientDiagnostics ClientDiagnostics = new(new TestClientOptions());
+        private static readonly ClientDiagnostics ClientDiagnostics = new(new TestClientOptions(), true);
```

Comment:
I think many of these tests are for HLC OperationInternals, so we shouldn't just update the ClientDiagnostics to only represent DPG.

Reply:
Does that mean we separate these tests out of this test class and put it into another one only for DPG?